tests/test_gndtruth.bats

```diff
@@ -0,0 +1,122 @@
+#          Note: Tested is occurrence of the gndtruth_lane_sensor              #
+################################################################################
+
+fn_run_and_test_compression() {
+    fn_cleanup "$1"
+    cloe_engine run test_gndtruth_smoketest-$2.json
+    file /tmp/cloe_gndtruth.$1 | grep "$3"
+}
+
+################################################################################
+# fn_unpack_gz                                                                 #
+################################################################################
+# Unpacks a gzip file                                                          #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: gzip file /tmp/cloe_gndtruth.$1 exists                        #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 is unpacked                           #
+################################################################################
+
+fn_unpack_gz() {
+    gunzip /tmp/cloe_gndtruth.$1
+}
+
+################################################################################
+# fn_unpack_bz2                                                                #
+################################################################################
+# Unpacks a bzip2 file                                                         #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: bzip2 file /tmp/cloe_gndtruth.$1 exists                       #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 is unpacked                           #
+################################################################################
+
+fn_unpack_bz2() {
+    bunzip2 /tmp/cloe_gndtruth.$1
+}
+
+################################################################################
+# fn_test_gndtruth                                                             #
+################################################################################
+# Minimal test for properties of a gndtruth file                               #
+# ---------------------------------------------------------------------------- #
+#   Parameter 1: extension of the filename (after /tmp/cloe_gndtruth.)         #
+#  Precondition: bzip2 file /tmp/cloe_gndtruth.$1 exists                       #
+# Postcondition: /tmp/cloe_gndtruth.*.$1 was tested                            #
+#          Note: Tested is occurrence of the gndtruth_lane_sensor              #
+#          Note: Tested is occurrence of the default_world_sensor              #
+#          Note: EXPECTED_CYCLE_COUNT := 1s / 20ms/Cycle - 1 Cycle             #
+################################################################################
+
+fn_test_gndtruth() {
+    EXPECTED_CYCLE_COUNT=49;
```

Comment:
Use `local` variables here and then make the identifiers, lowercase. See `setup_bats.bash` for examples.